src/modules/menu/components/settings/MenuSettings.vue

```diff
@@ -1,15 +1,32 @@
 <script setup>
+import { computed } from 'vue'
+import { useStore } from 'vuex'
+
 import LangSwitchToolbar from '@/modules/i18n/components/LangSwitchToolbar.vue'
 import LinksToolbar from '@/modules/menu/components/settings/LinksToolbar.vue'
+import MapFooterAppVersion from '@/utils/components/AppVersion.vue'
```

Comment:
I would import `AppVersion` here, unless there is something I'm completely missing.